### PR TITLE
Visitor sees a top story on homepage

### PR DESCRIPTION
**Pull request by Fraseer**
PT Link: https://www.pivotaltracker.com/story/show/180060761

## User Story
As a visitor,
In order to know what stories are popular,
I would like to have a top story on my home page

#Changes Proposed
- Create `Top Story` section
- Update fixture file to have top story attr
- Use top story attr to set the homepage top story

## Sreenshots

image
image

### CODE REVIEW

File: cypress/integration/userCanViewACollectionOfArticles.feature.js [hunk 1]
```diff
@@ -25,17 +46,8 @@ describe("A collection of articles is displayed in the main page", () => {
...
-        cy.get("[data-cy=created_at]").should(
-          "contain",
-          "2021-10-05"
-        );
-        cy.get("[data-cy=updated_at]").should(
-          "contain",
-          "2021-10-05"
-        );
+        cy.get("[data-cy=created_at]").should("contain", "2021-10-05");
+        cy.get("[data-cy=updated_at]").should("contain", "2021-10-05");
```
Comment (maxarvid) [inline cypress/integration/userCanViewACollectionOfArticles.feature.js]: Is this line doubled?

File: src/components/Header.jsx [hunk 2]
```diff
@@ -31,22 +32,36 @@ const Header = () => {
...
-      <Menu.Item
-        id="home"
-        name="home"
-        as={Link}
-        to={{ pathname: "/" }}
-        data-cy="home"
-      >
-        News In Progress
-      </Menu.Item>
+    <div>
```
Comment (maxarvid) [inline src/components/Header.jsx:35]: Is this `div` tag needed?

File: src/components/TopStory.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { Grid, Header, Item, Image } from "semantic-ui-react";
+
+const TopStory = () => {
+  const { articles } = useSelector((state) => state);
+
+  let topStory = articles.find(({ top_story }) => top_story === true);
```
Comment (maxarvid) [inline src/components/TopStory.jsx:8]: Nice use of `.find`!

File: src/styles/index.css [hunk 4]
```diff
@@ -20,7 +20,7 @@ p {
 
 .body {
   font-size: 1em;
-  text-align: justify;
+  /* text-align: justify; */
```
Comment (maxarvid) [inline src/styles/index.css]: Remove comment?

**Review by maxarvid — COMMENTED** [review]
Left a few comments

File: src/components/MostRead.jsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+          News Story Title
+        </List.Item>
+        <List.Item data-cy="story-4" as="li">
+          News Story Title
+        </List.Item>
+        <List.Item data-cy="story-5" as="li">
+          News Story Title
+        </List.Item>
+      </List>
+    </Grid.Column>
```
Comment (dorianbuck) [inline src/components/MostRead.jsx]: Hard coded?

File: src/locales/en.js [hunk 6]
```diff
@@ -1,11 +1,13 @@
 const en = {
   translation: {
-    categories: "Categories",
+    categories: "Menu",
```
Comment (dorianbuck) [inline src/locales/en.js:3]: This seems more newspaper/newsite 👍🏽

File: src/styles/index.css (hunk 4 above)
Comment (dorianbuck) [inline src/styles/index.css]: comment?

**Review by dorianbuck — COMMENTED** [review]
These are nice changes! Left some small comments

File: cypress/integration/userCanViewACollectionOfArticles.feature.js (hunk 1 above)
Comment (Fraseer) [reply cypress/integration/userCanViewACollectionOfArticles.feature.js]: removed

File: src/components/Header.jsx (hunk 2 above)
Comment (Fraseer) [reply src/components/Header.jsx:35]: now that the language moves outside the header on mobile we do

File: src/components/MostRead.jsx (hunk 5 above)
Comment (Fraseer) [reply src/components/MostRead.jsx]: Plan is to have it dynamically show the most viewed articles, but that has not been implemented yet. This is still needed though to show the layout of the website

File: src/components/Header.jsx (hunk 2 above)
Comment (maxarvid) [reply src/components/Header.jsx:35]: Roger roger.

**Review by maxarvid — APPROVED** [review]
Super!

**Review by dorianbuck — APPROVED** [review]

**Pull request merged**